Manifest.txt

```diff
@@ -158,8 +158,10 @@ bundler/lib/bundler/templates/newgem/newgem.gemspec.tt
 bundler/lib/bundler/templates/newgem/rspec.tt
 bundler/lib/bundler/templates/newgem/spec/newgem_spec.rb.tt
 bundler/lib/bundler/templates/newgem/spec/spec_helper.rb.tt
-bundler/lib/bundler/templates/newgem/test/newgem_test.rb.tt
-bundler/lib/bundler/templates/newgem/test/test_helper.rb.tt
+bundler/lib/bundler/templates/newgem/test/minitest/newgem_test.rb.tt
```

Comment:
L161-L164 are accidentally additions?

Reply:
I got it. But We shouldn't bundle without Bundler 2.1 branch.

Reply:
I don't think so, but there's maybe an issue here.

@simi bumped the vendored version of bundler to fix a testing issue, but after a second thought, we don't want to do that because that would mean including an undefined bundler version together with rubygems.

So, I'm not really sure, perhaps we can skip the bundler test suite on the stable branch.

Reply:
Or update the submodule to the proper ref dinamically during CI, not in source control.

Reply:
I can also prepare patch to apply on CI only to vendored bundler .

Reply:
Yeah, that too. Although @hsbt doesn't like `.patch` files :sweat_smile:

Reply:
Or remove `head` from CI of RG 3.1 branch. RG 3.1 support under the Ruby 2.7, not 2.8/3.0.

Reply:
I think it was not head failing, but the regular suite against the vendored bundler. But I agree with removing head anyways.

Reply:
@simi Are you still planning to work on this?

Reply:
yup, if this is the way for new 3.1 release, I'll try to make this green

Reply:
I'm not sure how @hsbt wants to handle it, but having a green 3.1 stable branch feels good anyways :man_shrugging: